src/mono/mono/mini/mini-ops.h

```diff
@@ -493,6 +493,11 @@ MINI_OP(OP_RBGT_UN, "r4_bgt_un", NONE, NONE, NONE)
 MINI_OP(OP_RBLE_UN, "r4_ble_un", NONE, NONE, NONE)
 MINI_OP(OP_RBLT_UN, "r4_blt_un", NONE, NONE, NONE)
 
+MINI_OP(XBINOP_FORCEINT_AND,    "bitwise_and", IREG, IREG, IREG)
+MINI_OP(XBINOP_FORCEINT_OR,     "bitwise_or", IREG, IREG, IREG)
```

Comment:
These are not actually opcodes, so they shouldn't be in this file.

Reply:
Created enum in mini.h instead.